shelley/chain-and-ledger/formal-spec/delegation.tex

```diff
@@ -214,6 +217,13 @@ \subsection{Delegation Transitions}
     \end{itemize}
 \end{itemize}
 
+The operational certificates $\var{cs}$ in the stake pool state are a tool to
+prevent nodes from rejecting blocks signed by a new hot staking key.
```

Comment:
We could reword this as something like "The operational certificates counters $\var{cs}$ in the stake pool state are a tool to ensure that blocks containing outdated certificates are rejected"